s3/src/main/scala/monix/connect/s3/S3.scala

```diff
@@ -885,6 +884,193 @@ trait S3 { self =>
     } yield s3Object
   }
 
+  private def listNHelper(
```

Comment:
would you mind moving this method to the companion objects for `ListObjectsObservable`?

Reply:
moved